ProtocolHandler.pm

```diff
@@ -2,7 +2,7 @@ package Plugins::GoogleMusic::ProtocolHandler;
 
 use strict;
 use warnings;
-use base qw(Slim::Player::Protocols::HTTP);
+use base qw(Slim::Player::Protocols::HTTPS);
```

Comment:
Wrapping this in a conditional to check the version I think should work:
```
if ( Slim::Utils::Versions->compareVersions($::VERSION, '7.9.0') < 0 ) {
    eval "use base qw(Slim::Player::Protocols::HTTP)";
} else {
    eval "use base qw(Slim::Player::Protocols::HTTPS)";
}
```
Only tested on 7.9.1.

Reply:
I've made my own fork of this repo and commit the change in https://github.com/davewongillies/squeezebox-googlemusic/commit/69529b7a1b420fd0c5ef2fc93e129fef07d8d204

And have also pushed a new version for to my fork of the repo here: https://davewongillies.github.io/squeezebox-googlemusic/repository/repo.xml

Reply:
It is worth submitting a PR here to keep everything in one place?

Pretty sure mine didn't get merged due to it breaking <= 7.8, but as your fix solves this ping @askvictor and see if he will add you to the repo :)

Reply:
@digiltd I'm happy to raise another PR with the fix.

Reply:
@digiltd do you know if the `streamURL` change mentioned in https://github.com/squeezebox-googlemusic/squeezebox-googlemusic/issues/14 is required as well? I'll test it out on <= 7.8 though (I tested the plugin to make sure it didn't have any errors but didn't test playback).

Reply:
no idea sorry, i think #14 talks about an either or situation

a working pr is always a good pr :)

Reply:
So I've raised https://github.com/squeezebox-googlemusic/squeezebox-googlemusic/pull/25 as a replacement for this PR. If you've got the chance, I'd appreciate if you could test out my changes.